lib/ansible/module_utils/compat/paramiko.py

```diff
@@ -11,7 +11,12 @@
 
 try:
     with warnings.catch_warnings():
-        warnings.filterwarnings('ignore', message='Blowfish has been deprecated', category=UserWarning)
+        # Blowfish is deprecated in cryptography, but still used by paramiko versions older than 2.9.5.
+        # See: https://github.com/paramiko/paramiko/pull/2039
+        warnings.filterwarnings('ignore', message='Blowfish has been ', category=UserWarning)
+        # TripleDES is deprecated in cryptography, but still used by paramiko versions older than 3.3.2 and 3.4.1.
```

Comment:
Being pedantic it hasn't been deprecated, it's just moved to a new namespace and a newer version of Cryptography will remove the old import path in the new version.

Reply:
I revised the comments to clarify what is deprecated.